rastervision_pytorch_learner/rastervision/pytorch_learner/dataset/dataset.py

```diff
@@ -47,6 +47,8 @@ def __init__(self,
             self.normalize = False
```

Comment:
To keep it consistent with object detection, I would just add the check here and set `self.to_pytorch = False` and then modify `regression_transformer()` in `transform.py` to return `x` and `y` in the correct form.

On second thought, perhaps the `y = torch.from_numpy(y).long()` in `__getitem__` is perhaps too strong an assumption. Maybe it should just be `y = torch.from_numpy(y)` and we should expect the transform functions to supply a `y` with the correct data type.